packages/docusaurus-theme-bootstrap/src/types.d.ts

```diff
@@ -92,6 +92,18 @@ declare module '@theme/Tabs' {
   export default Tabs;
 }
 
+declare module '@theme/ThemedImage' {
+  import type {ComponentProps} from 'react';
+
+  export type Props = {
+    readonly lightSrc: string;
+    readonly darkSrc?: string;
+  } & ComponentProps<'img'>;
+
+  const ThemedImage: () => JSX.Element;
```

Comment:
despite other examples not declaring components props properly,  I think we should do this (cf TabItem, Layout).

We'll make sure all comps have `props: Props` over time

Reply:
I'm not against this change, but this sound like a feature request or a bug fix request, so I think you should handle this for all theme components in the separate PR.